data/D2D/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Prime",
+  "symbol": "D2D",
+  "decimals": 18,
+  "description": "A DAO collective focused on researching and building next-generation coordination tools for decentralized organizations.",
+  "website": "https://www.prime.xyz/",
+  "twitter": "@PrimeDAO_",
+  "tokens": {
+    "ethereum": {
+      "address": "0x43D4A3cd90ddD2F8f4f693170C9c8098163502ad"
+    },
+    "base": {
+      "address": "0x91e1d2b9b0d9afcf1f024df09a68799fb200bc63"
+    },
+    "optimism": {
+      "address": "0x03e5b5930f3a42b56af4a982b10e60957bdd2f61"
+    },
```

Comment:
json failing to validate, I think you need to drop this comma

Reply:
```suggestion
    }
```